components/brave_wallet/common/BUILD.gn

```diff
@@ -0,0 +1,7 @@
+import("//mojo/public/tools/bindings/mojom.gni")
```

Comment:
same as brave_wallet/browser. A component must either not have common/browser/renderer/etc... or only have them. It can't have files in both the root and brower/common/etc...

Reply:
everything has been moved to browser folder